cdap-hbase-spi/src/main/java/co/cask/cdap/spi/hbase/HBaseDDLExecutor.java

```diff
@@ -82,7 +94,11 @@ void createTableIfNotExists(TableDescriptor descriptor, @Nullable byte[][] split
   void disableTableIfEnabled(String namespace, String name) throws IOException;
 
   /**
-   * Modify the specified table. The table must be disabled.
+   * Modify the specified table. Changing the dataset specs results into updating the
```

Comment:
This is called when an HBase backed dataset has its properties modified. In order to modify the HBase table, CDAP first calls disableTableIfEnabled, then calls this method, then calls enableTableIfDisabled.